src/components/organisms/Navbar/Navbar/Navbar.md

```diff
@@ -1,27 +1,374 @@
 ### Summary
 
-`<Navbar>` renders the base navigation bar layout, creating **right**, **left**, and **center** to render the contents you like.
+Navbar will render an array of links provided using a dropdown which works on click and a link which is a wrap around our [`<Link />`](/#/Components/Atoms/Link) atom, decorated with tow extra props: `active` and `withChevron`. You can opt out displaying the CTA using `withCTA` prop.
 
-It's meant to be used with other navbar components like [`<Navbar.Link />`](#/Components/Organisms/Navbar/NavbarLink) and [`<Navbar.Dropdown />`](#/Components/Organisms/Navbar/NavbarDropdown)
+⚠️ The menu animates when scrolling. For a better preview view [Default example](#/Components/Organisms/Navbar/Navbar/1)
 
 ### Examples
 
-- White theme
+- Default theme
 
-```ts { "props": { "style": { "transform": "translate3d(0, 0, 0)", "backgroundColor": "#00B9A7", "border": "2px solid #efefef" } } }
-import { Navbar, Link } from '@zopauk/react-components';
+```ts { "props": { "style": { "transform": "translate3d(0, 0, 0)", "backgroundColor": "#FFFFFF", "overflow": "hidden", "height": "500px" } } }
+import { Navbar } from '@zopauk/react-components';
 
-<Navbar left={<Link>left</Link>} center={<Link>center</Link>} right={<Link>right</Link>} />;
+const NAV_ITEMS = [
+  {
+    label: 'Borrow',
+    href: '/loans',
```

Comment:
To avoid confusion, it's probably best to remove it the `href` from the dropdown opener. Same thing below